apps/daimo-mobile/src/logic/nfcListen.ts

```diff
@@ -0,0 +1,69 @@
+import NfcManager, {NfcTech} from 'react-native-nfc-manager';
+import { MAX_NONCE_ID_SIZE_BITS } from "@daimo/userop";
+import { Hex } from "viem";
+import { generatePrivateKey } from "viem/accounts";
+import { useEffect } from "react";
+import { useNavigation } from "@react-navigation/native";
+import type { NativeStackNavigationProp } from "@react-navigation/native-stack";
+import { HomeStackParamList } from "../view/shared/nav";
+import { Recipient, getRecipient } from "../sync/recipients";
+
+NfcManager.start();
+
+
+async function readNdef(nav: NativeStackNavigationProp<HomeStackParamList>) {
+    try {
+      // register for the NFC tag with NDEF in it
+      try {
+        await NfcManager.requestTechnology(NfcTech.IsoDep);
+      } catch (ex) {
+      }
+      // the resolved tag object will contain `ndefMessage` property
+      const tag = await NfcManager.getTag();
+      var tesp = await NfcManager.isoDepHandler.transceive([0x00, 0xA4, 0x04, 0x00, 0x07, 0xD2, 0x76, 0x00, 0x00, 0x85, 0x01, 0x01, 0x00]);
```

Comment:
add link to wherever you got the constant from `// (nfc doc url)`

might consider `hexToBytes("0x00A4040007D276000085010100")`